src/narjure/memory_management/concept.clj

```diff
@@ -1,30 +1,80 @@
 (ns narjure.memory-management.concept
   (:require
+    [co.paralleluniverse.pulsar.actors
+     :refer [! spawn gen-server register! cast! Server self
+             shutdown! unregister! set-state! state whereis]]
     [narjure.actor.utils :refer [defactor]]
     [taoensso.timbre :as t])
   (:refer-clojure :exclude [promise await]))
 
-(declare concept task-req belief-req inference-req persistence-req exit)
+(defn debug [msg] (t/debug :concept msg))
 
-(defactor concept
-          "State is a map
-          {:name :budget :activation-level :belief-tab :goal-tab :task-bag :term-bag}
-          (this list may not be complete)."
-          {:task-msg            task-req
-           :belief-req-msq      belief-req
-           :inference-req-msq   inference-req
-           :persistence-req-msg persistence-req})
+(defn task-handler
+  ""
+  [from message]
+  ())
```

Comment:
What is the reason for empty parentheses inside functions? It looks weird. If it is a reminder about missed code it would be better to see `;todo` comment there.